workflow-templates/add-issue-to-a-board.yml

```
@@ -0,0 +1,17 @@
+# Run the workflow that adds newly created issues to a specific
+# GitHub Project
+
+name: Add newly created issues to the project board
```

```suggestion
name: Add newly created issues to the project
```

We're adding to a project in general.  A board is a view within a project.